### PR TITLE
Add 'How to Install Thumbor' blog post

### LINKED ISSUE #1148: blog: How to Install Thumbor
**Author**: @khempenius
**Reviewer**: @kaycebasques

**A one to two sentence description of your post**
Instructions on how to install Thumbor.

What are you planning to teach folks in this post?
How to install Thumbor.

**Target publish date:** `2019-08-14`

- [ ] Check this box if this is a hard deadline.

**Process**

- [x] 1. I've done my [proposal](https://docs.google.com/document/d/1SSq8tkwYfNn2bGbp_eqIlA6VsV-IE8q47PnSzUVUz3I/edit?usp=sharing)
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [x] 4. Draft approved — @kaycebasques LGTM
- [x] 5. I've submitted a pull request #1214
- [x] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by khempenius**
Fixes #1148

### CODE REVIEW

File: src/site/content/en/blog/install-thumbor/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+title: How to install the Thumbor image CDN
+subhead: Learn how to install the Thumbor image CDN on your own server.
+authors:
+  - katiehempenius
+date: 2019-07-29
+hero: hero.jpg
+alt: A pile of photos.
+description: |
+  Instructions on how to install Thumbor is an open-source image CDN and can be used for free to resize, compress, and transform images.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: Looks like you started writing one sentence and then finished with another: 

> Instructions on how to install Thumbor is an open-source image CDN

You can fix by deleting `Instructions on how to install`

File: src/site/content/en/blog/install-thumbor/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+  - images
+---
+
+Image CDNs make it easy to dynamically optimize the aesthetics and performance of your images. Unlike most image CDNs, Thumbor is  open-source and can be used for free to resize, compress, and transform images. It's suitable for production use; [Wikipedia](https://wikitech.wikimedia.org/wiki/Thumbor) and [Square](https://medium.com/square-corner-blog/dynamic-images-with-thumbor-a430a1cfcd87) both use Thumbor.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: Remove the extra space between `is` and `open-source`

> Thumbor is  open-source and can

File: src/site/content/en/blog/install-thumbor/index.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+## Install Thumbor Dependencies
+
+Update and upgrade Ubuntu's already-installed packages:
+
+```bash
+sudo apt-get update -y && sudo apt-get upgrade -y
+```
+
+Install pip, the package manager for Python. Later you'll install Thumbor with pip.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: Whenever mentioning a command line tool like `pip` style it as inline code, like this:

> Install `pip`, the package manager for Python. Later you'll install Thumbor with `pip`.

https://developers.google.com/style/command-line-terminology

File: src/site/content/en/blog/install-thumbor/index.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+```bash
+# image format packages
+sudo apt-get install -y libjpeg-dev libpng-dev libwebp-dev webp
+```
+
+## Install Thumbor
+
+Install Thumbor using pip.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: pip => \`pip\`

File: src/site/content/en/blog/install-thumbor/index.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+```bash
+sudo pip install thumbor
+```
+
+Note: Many Python developers use [virtualenv](https://pypi.org/project/virtualenv/) to manage their packages. For the sake of simplicity, these instructions do not use virtualenv. If you are installing Thumbor in a standalone environment, virtualenv is not necessary. If you choose to use virtualenv, note that Thumbor requires Python 2.7 and will not work with newer versions of pip (e.g. these instructions use pip 8.1.1).
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: Format pip and virtualenv as inline code

File: src/site/content/en/blog/install-thumbor/index.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+Note that this URL uses HTTP. Thumbor uses HTTP by default but can be [configured](https://thumbor.readthedocs.io/en/latest/image_loader.html) to use HTTPS.
+
+
+You should see an image that is 100 pixels wide by 100 pixels tall. Thumbor has taken the image `hero.jpg` and size specified in the URL string and served the result. You can replace the image in the URL string (i.e `https://web.dev/install-thumbor/hero.jpg`) with any other image (e.g. `https://your-site.com/cat.jpg`) and Thumbor will resize that image too. 
+
+
+## Use Thumbor
+
+You’ve successfully installed Thumbor. Next, learn about using Thumbor in the [Use Thumbor to Optimize Images](https://web.dev/codelab-thumbor) codelab.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: Use sentence case for the title. `Use Thumbor to Optimize Images` => `Use Thumbor to optimize images`

https://web.dev/handbook/grammar/#titles-and-headings

File: src/site/content/en/blog/install-thumbor/index.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+If you are using this guide to set up Thumbor in production, you may also want to check out the optional section below on systemd.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: systemd => \`systemd\`

File: src/site/content/en/blog/install-thumbor/index.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+You’ve successfully installed Thumbor. Next, learn about using Thumbor in the [Use Thumbor to Optimize Images](https://web.dev/codelab-thumbor) codelab.
+
+If you are using this guide to set up Thumbor in production, you may also want to check out the optional section below on systemd. 
+
+
+## Appendix: Configuring Systemd
+
+This step explains how to make sure that the Thumbor process keeps running, even after the VM has been restarted. This step is important for production sites, but optional if you’re just playing around with Thumbor.
+
+[Systemd](https://www.freedesktop.org/software/systemd/man/systemd.html) is the “system and service manager” for Linux operating systems. Systemd makes it easy to configure when services (processes) run.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: Systemd => \`systemd\`

(Keep it consistent with how you styled it above)

File: src/site/content/en/blog/install-thumbor/index.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+
+We will be configuring systemd to automatically start Thumbor on VM boot. If the VM is restarted, the Thumbor process will automatically restart as well. This is much more reliable than relying on user intervention to start Thumbor.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: We => You

https://developers.google.com/style/person

File: src/site/content/en/blog/install-thumbor/index.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+
+Navigate to the `/lib/systemd/system` directory. This directory contains the service files for systemd.
```
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: systemd => \`systemd\`

File: src/site/content/en/blog/install-thumbor/index.md (hunk 2 above)
Comment (kaycebasques) [inline src/site/content/en/blog/install-thumbor/index.md]: And link to Thumbor's site the first time that you mention it: http://thumbor.org/

**Comment by khempenius** [comment]
@kaycebasques fixed

File: src/site/content/en/blog/install-thumbor/index.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
+---
+title: How to install the Thumbor image CDN
+subhead: Learn how to install the Thumbor image CDN on your own server.
```
Comment (robdodson) [inline src/site/content/en/blog/install-thumbor/index.md]: Would it be possible to punch up the subhead a bit? I've been encouraging folks to use it as an opportunity to give the reader a bit more context on why they should care about the topic.

Comment (khempenius) [reply src/site/content/en/blog/install-thumbor/index.md]: Is this better?

Comment (robdodson) [reply src/site/content/en/blog/install-thumbor/index.md]: yep I think that's good. Thanks!

**Comment by kaycebasques** [comment]
LGTM thanks @khempenius

File: src/site/content/en/blog/install-thumbor/index.md [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+Verify that you’ve successfully configured `systemd` by running the `status` command.
+
+```bash
+systemctl status thumbor.service
+```
+
+If you’ve successfully set up thumbor.service to use `systemd`, the [status](https://www.freedesktop.org/software/systemd/man/systemctl.html#status%20PATTERN%E2%80%A6%7CPID%E2%80%A6%5D) should show that it is enabled and active.
+
+<figure class="w-figure  w-figure--center">
+  <img src="./systemd.jpg" alt="Systemctl displaying the status of Thumbor">
```
Comment (robdodson) [inline src/site/content/en/blog/install-thumbor/index.md]: I don't want to hold up the post but I was wondering if we might want to come back at some point and use a larger version of this image. The max width is ~800px. Currently at this size the image is a little hard to read.

cc @kaycebasques

File: src/site/content/en/blog/install-thumbor/index.md [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+`systemctl` is the utility used to manage `systemd`. Use the `start` command to start Thumbor.
+
+```bash
+sudo systemctl start thumbor.service
+```
+Note: If Thumbor is currently running, you should stop it before attempting to start Thumbor using `systemctl`.
+
+Next, “enable” Thumbor. This means that Thumbor will automatically start on boot.
```
Comment (robdodson) [inline src/site/content/en/blog/install-thumbor/index.md]:
```suggestion
Next, "enable" Thumbor. This means that Thumbor will automatically start on boot.
```

**Comment by robdodson** [comment]
@khempenius mind fixing the markdown linter errors?
https://travis-ci.org/GoogleChrome/web.dev/builds/569969441?utm_source=github_status&utm_medium=notification

cc @kaycebasques wanted to flag this as something to watch out for. We recently added markdown linting which will cause Travis to fail.

**Comment by khempenius** [comment]
@robdodson Fixed 👍

**Comment by robdodson** [comment]
thank you! 🎉

**Pull request merged**